Yosemite/YosemiteTests/Stores/OrderStoreTests.swift

```diff
@@ -906,6 +912,43 @@ private extension OrderStoreTests {
         return [coupon1, coupon2]
     }
 
+    func sampleOrderTaxLines() -> [Networking.OrderTaxLine] {
+        [
+            .init(taxID: 1330,
+                  rateCode: "US-NY-STATE-2",
+                  rateID: 6,
+                  label: "State",
+                  isCompoundTaxRate: true,
+                  totalTax: "7.71",
+                  totalShippingTax: "0.00",
+                  ratePercent: 4.5,
+                  attributes: [])
+        ]
+    }
+
+    func sampleOrderTaxLinesMutated() -> [Networking.OrderTaxLine] {
+        [
+            .init(taxID: 1330,
+                  rateCode: "US-NY-STATE-2",
+                  rateID: 6,
+                  label: "State",
+                  isCompoundTaxRate: true,
+                  totalTax: "55",
+                  totalShippingTax: "0.00",
+                  ratePercent: 5.5,
+                  attributes: []),
+            .init(taxID: 124,
+                  rateCode: "",
+                  rateID: 2,
+                  label: "Central",
+                  isCompoundTaxRate: true,
+                  totalTax: "2.5",
+                  totalShippingTax: "3",
+                  ratePercent: 2.5,
+                  attributes: [])
```

Comment:
Similar to previous comments - these can be replaced with `.fake()` and `.copy()` for simplicity.

Reply:
Done in - https://github.com/woocommerce/woocommerce-ios/pull/5931/commits/cb3593fd15c2285012ed43cb4657b9cdf203bdf1